sagemaker-featurestore/README.md

```diff
@@ -8,11 +8,21 @@ This notebook requires these data sets in `./data/`:
 * `feature_store_introduction_customer.csv`
 * `feature_store_introduction_orders.csv`
 
-This notebook requires these images in `./images/:
+This notebook requires these images in `./images/`:
 
 * `feature-store-policy.png`
 * `feature_store_data_ingest.svg`.
 
+## Feature Store Encryption with KMS key
+In `feature_store_kms_encryption.ipynb` we demonstrate how to encrypt data in your online or offline store using KMS key and how to verify that your KMS key is being used for data encryption. 
```

Comment:
Add comma after "feature_store_kms_encryption.ipynb"
Add "a" between "using" and "KMS"